GetandTake/Services/Concrete/CategoryImageManager.cs

```diff
@@ -0,0 +1,33 @@
+﻿using GetandTake.Core.Helper;
+using GetandTake.DataAccess.Repositories.Abstract;
+using GetandTake.Models;
+using GetandTake.Services.Abstract;
+
+namespace GetandTake.Services.Concrete;
+
+public class CategoryImageManager : ICategoryImageService
+{
+    private readonly ICategoryImageRepository _categoryImageRepository;
+
+    public CategoryImageManager(ICategoryImageRepository categoryImageRepository)
+    {
+        _categoryImageRepository = categoryImageRepository;
+    }
+
+    public async Task Add(IFormFile file,int id)
```

Comment:
Please, separate your args with spaces after comma

Reply:
this one deleted